archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/ScriptLoader.java

```diff
@@ -319,6 +330,26 @@ void processInput() {
             addChild(nextState, Input.builder(info, kind));
         }
 
+        void processValidation() {
+            Block.Kind kind = blockKind();
+            if (kind == VALIDATION) {
```

Comment:
Avoid the static import here. No need to preserve the same style as the switch statements ;)